src/compas/geometry/shapes/_shape.py

```diff
@@ -3,12 +3,77 @@
 from __future__ import division
 
 import abc
-from compas.geometry import Primitive
+from ..geometry import Geometry
 
 
-class Shape(Primitive):
+class Shape(Geometry):
```

Comment:
technically, this is a breaking change, but I guess it's not very problematic. However, it would be fair to add a bigger warning/notice in the changelog indicating that if someone depends on this fact, they need to update.

Reply:
there is this in the log
```
* Changed base class of `compas.geometry.Primitive` and `compas.geometry.Shape` to `compas.geometry.Geometry`.
```